backend/api/Areas/Keycloak/Profiles/User/Resolvers/UpdateRoleToEntityResolver.cs

```diff
@@ -0,0 +1,34 @@
+using System.Collections.Generic;
+using System.Linq;
+using AutoMapper;
+using Entity = Pims.Dal.Entities;
+using Model = Pims.Api.Areas.Keycloak.Models.User;
+
+namespace Pims.Api.Areas.Keycloak.Profiles.User.Resolvers
+{
+    /// <summary>
+    /// UpdateRoleToEntityResolver class, provides automapper configuration to convert entiy agencies to model agencies.
```

Comment:
should be converting entity roles to model roles I believe.

Reply:
This copies a Role Model into an Entity.